packages/block-editor/src/components/inserter/menu.js

```diff
@@ -380,9 +426,13 @@ export default compose(
 		}
 		const destinationRootBlockName = getBlockName( destinationRootClientId );
 
+		let items = getInserterItems( destinationRootClientId );
+		const discoverBlocks = getDiscoverBlocks();
```

Comment:
If I undertand properly, this triggers the API to fetch everything available when you open the inserter. I was expecting something that gets called when you type and no result is found in the installed blocks. I guess this is just an early prototype?

Reply:
Yes, this just an early prototype. I will be making changes to only call when no results are found in the installed blocks.